.pre-commit-config.yaml

```diff
@@ -0,0 +1,16 @@
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v3.2.0
+    hooks:
+      - id: check-yaml
+        exclude: conda/meta.yaml
+
+  - repo: https://github.com/psf/black
+    rev: 19.10b0
```

Comment:
This will probably increase the amount of work needed to upgrade the version of black... but I think it's fine... Can't think of a better way to handle it.